rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/DefaultInfrastructureManager.java

```diff
@@ -88,18 +88,18 @@ public void acquireAllNodes() {
     }
 
     /**
-     * Remove the node from underlying infrastructure. Terminates proactive
-     * runtime if there is no more nodes.
-     * 
-     * @param node
-     *            node to release
-     * @throws RMException
-     *             if any problems occurred
+     * {@inheritDoc}
      */
     @Override
-    public void removeNode(Node node) throws RMException {
+    public void removeNode(Node node, boolean dueToDownNode) throws RMException {
```

Comment:
It would be clearer to make two methods instead of one. removeNode and notifyDownNode, unless there is a reason for keeping a single method I don't see.